### PR TITLE
Update rpms for 2.0.0

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

**Review by jwendell — CHANGES_REQUESTED** [review]
Remove GN from this PR.
See https://github.com/maistra/rpms/pull/106

File: istio-proxy/update.sh [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
 
 function usage() {
-    echo "Usage: $0 [-i <SHA of proxy>]"
+    echo "Usage: $0 [-p <SHA of proxy>] [-g <SHA of gn>]"
     echo
     exit 0
 }
 
-while getopts ":i:" opt; do
+while getopts ":p:g:" opt; do
```
Comment (jwendell) [inline istio-proxy/update.sh]: You can remove all gn, can't you?

File: istio-proxy/sources [hunk 2]
```diff
@@ -1 +1,2 @@
-eb51ab9f9752064487422401d6985923  proxy-327e423863f85c4709b1734029898d08a4e2dd48.tar.gz
+dd596846759c29fa9080edcc6491442a  proxy-84e3195d7892961771ba610d1012b3c0fc912783.tar.gz
+1d1817eda3a3ee5fc1dfeb2a76e8db51  gn-1d1817eda3a3ee5fc1dfeb2a76e8db51.tar.gz
```
Comment (jwendell) [inline istio-proxy/sources]: remove gn?

File: istio-proxy/istio-proxy.spec [hunk 3]
```diff
@@ -52,11 +51,16 @@ BuildRequires:  platform-python-devel
...
 Source0:        proxy-%{git_commit}.tar.gz
 
+ExclusiveArch:  x86_64
+
 %description
 The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
 
 %prep
 %setup -q -n proxy-%{git_commit}
+%setup -q -D -T -n proxy-%{git_commit} -a 10
```
Comment (jwendell) [inline istio-proxy/istio-proxy.spec]: is this right? What's with `-a 10`?

File: istio-proxy/istio-proxy.spec [hunk 4]
```diff
@@ -52,11 +51,16 @@ BuildRequires:  platform-python-devel
...
+pushd maistra/vendor/envoy
+popd
```
Comment (jwendell) [inline istio-proxy/istio-proxy.spec]: Why this empty push?

File: istio-proxy/istio-proxy.spec [hunk 5]
```diff
@@ -40,7 +40,6 @@ BuildRequires:  gcc-c++
 BuildRequires:  make
 BuildRequires:  patch
 BuildRequires:  xz
-BuildRequires:  git
 BuildRequires:  golang
```
Comment (jwendell) [inline istio-proxy/istio-proxy.spec]: You probably have to add gn here...

**Review by jwendell — CHANGES_REQUESTED** [review]
I'd say to try a COPR build and make sure it's building. I think some changes are necessary to proxy .spec file.

**Comment by brian-avery** [comment]
Aside from the gn magic in this PR, these are the same spec files we are using to build product. I can kick off an RPM build to see if it works.

File: istio-proxy/istio-proxy.spec (hunk 3 above)
Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: Not actually sure what this is doing, but this is how we are doing it in product. This PR just synchronizes the two. Let me look into that.

Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: Option -a (which stands for after) expands those sources after entering. Their arguments are source numbers from the spec file preamble.

From: https://rpm-packaging-guide.github.io/

Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: GN was labeled as source 10. Based on that, I think this was expanding that.

File: istio-proxy/istio-proxy.spec (hunk 4 above)
Comment (brian-avery) [reply istio-proxy/istio-proxy.spec]: I'm not actually sure. Perhaps we should remove it from product too?

**Comment by brian-avery** [comment]
Operator, IOR, and CNI built. Looks like proxy and istio both have issues.

File: istio-proxy/istio-proxy.spec [hunk 6]
```diff
@@ -13,14 +13,14 @@
...
+%global git_commit 29093158179dc65392f41e664617e3574e56f124
 %global shortcommit  %(c=%{git_commit}; echo ${c:0:7})
 
 # https://github.com/maistra/proxy
 %global provider        github
 %global provider_tld    com
-%global project         maistra
-%global repo            proxy
+%global project         brian-avery
+%global repo            proxy-1
```
Comment (dgn) [inline istio-proxy/istio-proxy.spec]: these two lines seem to be accidental commits?

File: istio-proxy/istio-proxy.spec [hunk 7]
```diff
@@ -3,7 +3,7 @@
 # Build with debug info rpm
 %global with_debug 0
 # Run unit tests
-%global with_tests 1
+%global with_tests 0
```
Comment (dgn) [inline istio-proxy/istio-proxy.spec:6]: was this intended?

**Review by dmitri-d — APPROVED** [review]

File: istio-operator/istio-operator.spec [hunk 8]
```diff
@@ -13,7 +13,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit 9f557cabee101f05679493358a400061108dfe23
+%global git_commit cf6c6a13c3b37f48853a5a3b055e6e82caffeebb
```
Comment (jwendell) [inline istio-operator/istio-operator.spec]: This is outdated

**Review by jwendell — COMMENTED** [review]
LGTM, just the operator SHA that needs to be updated

File: istio-operator/istio-operator.spec (hunk 8 above)
Comment (brian-avery) [reply istio-operator/istio-operator.spec]: Updated the SHA and did a test build with it.

**Review by jwendell — APPROVED** [review]

File: istio-proxy/istio-proxy.spec (hunk 7 above)
Comment (brian-avery) [reply istio-proxy/istio-proxy.spec:6]: Doing this following product. We have additional work that we need to do to add dependencies if we want to run the tests.

**Pull request merged**